Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts

```diff
@@ -28,7 +28,7 @@ app.use(function (req, res, next) {
 // development error handler
 // will print stacktrace
 if (app.get('env') === 'development') {
-    app.use((err: any, req, res, next) => {
+    app.use((err, req, res, next) => { // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
I would remove `next` instead. I don't know if the rule will still be required for `req`. I don't think so.

Reply:
"Error-handling middleware always takes four arguments. You must provide four arguments to identify it as an error-handling middleware function. Even if you don’t need to use the next object, you must specify it to maintain the signature." from [Express documentation](https://expressjs.com/en/api.html)

Reply:
I found a bit weird the pattern but fair enough, disabling the eslint rule should do it. Good catch.